lienflash/lienflash-fe
# Fix errors, error handling, and improved testing

#### What’s this PR do?
- Fixes JobDetails bug
- Fix tests to cover changes made in code
- Small adjustments for clarity based on feedback
- Restructure folders to be in containers if they use the store
- Add clearing error messages to be tied to clicking to go home

#### Where should the reviewer start?
- all over the place

#### How should this be manually tested?
- in terminal, run npm test to see all tests are passing
- run npm test -- --coverage if you want to see current coverage percentage
- run npm start to see changes on the browser
